<launch>

<node name="groundstation_node" pkg="robot_groundstation" type="groundstation_node" output="screen" >
<param name="station_mode" type="string" value="5G1" />
</node>  

</launch>
